build.gradle

```diff
@@ -38,21 +38,7 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
-dokka() {
-    outputFormat = "html"
-    outputDirectory = "$buildDir/kdoc"
-    subProjects = ["core", "push"]
-
-    configuration {
-        moduleName = 'data'
-
-        includeNonPublic = true
-        skipDeprecated = false
-        reportUndocumented = true
-        skipEmptyPackages = true
-
-        includes = ["README.md"]
-
-        jdkVersion = 11
-    }
+dokkaHtmlMultiModule {
+    outputDirectory = file("${buildDir}/../docs")
+    includes.from("README.md")
```

Comment:
Just curious what the includes does ?

Reply:
I think its like `include` in php, it just appends that file's content into the home page of the output? something like that.